dot/peerset/handler.go

```diff
@@ -30,80 +30,79 @@ func NewPeerSetHandler(cfg *ConfigSet) (*Handler, error) {
 	}, nil
 }
 
+// SetReservedOnlyPeer not yet implemented
+func (h *Handler) SetReservedOnlyPeer(setID int, peers ...peer.ID) {
```

Comment:
I feel like these functions should return an error now, and the caller should log it.